architecture/rhacs-cloud-service-service-description.adoc

```diff
@@ -0,0 +1,183 @@
+:_content-type: ASSEMBLY
+[id="rhacs-cloud-service-service-description"]
+= {product-title-managed-short} service description
+include::modules/common-attributes.adoc[]
+:context: access-policy-service-description
+
+toc::[]
+
+[id="introduction-to-rhacs_{context}"]
+== Introduction to {product-title-short}
+
+{rh-rhacs-first} is an enterprise-ready, Kubernetes-native container security solution that helps you build, deploy, and run cloud-native applications more securely.
+
+{rh-rhacscs-first} provides Kubernetes-native security as a service.
+With {product-title-managed-short}, Red Hat maintains, upgrades, and manages your Central services.
+
+Central services include the user interface (UI), data storage, {product-title-short} application programming interface (API), and image scanning capabilities.
+You deploy your Central service through the link:https://console.redhat.com/[Red Hat Hybrid Cloud Console.] When you create a new Central instance, Red Hat creates your individual control plane for {product-title-short}.
+
+{product-title-managed-short} allows you to secure self-managed clusters that can reach out to a Central instance.
+The clusters you secure, called Secured Clusters, are managed by you, and not by Red Hat.
+Secured Cluster services include optional vulnerability scanning services, admission control services, and data collection services used for runtime monitoring and compliance.
+You install Secured Cluster services on any OpenShift or Kubernetes cluster you want to secure.
+
+[id="billing_{context}"]
+== Billing
+
+Customers can purchase a {product-title-managed-short} subscription on the Amazon Web Services (AWS) marketplace.
+The service cost is charged hourly per secured core ^[1]^.
+
+.Example
+====
+If you have established a connection between two secured clusters, each with 5 identical nodes with 8 vCPUs (such as Amazon EC2 m7g.2xlarge), the total number of secured cores is 80 (2 x 5 x 8 = 80).
```

Comment:
```suggestion
If you have established a connection to two secured clusters, each with 5 identical nodes with 8 vCPUs (such as Amazon EC2 m7g.2xlarge), the total number of secured cores is 80 (2 x 5 x 8 = 80).
```

Context: in this case, both secured clusters are connected to the ACSCS Central.